packages/react-instantsearch/src/connectors/connectHierarchicalMenu.test.js

```diff
@@ -130,7 +130,7 @@ describe('connectHierarchicalMenu', () => {
     expect(props).toBe(null);
   });
 
-  it('calling refine updates the widget\'s state', () => {
+  it('calling refine updates the widget\'s searchState', () => {
```

Comment:
I think search state is better (disregard this comment if you disagree with my previous remark :) )